docs/codeql/reusables/extractors.rst

```diff
@@ -5,15 +5,15 @@
    * - Language
      - Identifier
    * - C/C++ 
-     - ``cpp``
+     - ``c-cpp``
```

Comment:
@henrymercer can you check this file's changes? I think we're not changing the extractor names, but as a language name this works. @saritai where is this rst file is being used in the docs?

Reply:
That's correct — the extractor names are not changing, to reduce risk and keep things like results in the same place.  Instead we're just adding `c-cpp`, `java-kotlin`, and `javascript-typescript` as aliases that'll work in code scanning workflows and `--language <>` CLI arguments.

Would it make sense to add an extra column to the table for aliases?

Reply:
actually - this reusable doesn't seem to be used anywhere in the docs, so this may not matter! I don't see it mentioned anywhere. I just updated it regardless but I think I can just remove the changes and leave as-is (maybe we could even delete this but not sure)